### PR TITLE
[buildsystem] Install debug packages in syncd when INSTALL_DEBUG_TOOLS=y

### PR DESCRIPTION
**Pull request by stepanblyschak**
Signed-off-by: Stepan Blyschak <stepanb@mellanox.com>

**- What I did**
Install debug packages when INSTALL_DEBUG_TOOLS=y, the same way as with swss docker image;

**- How to verify it**
Installed docker syncd(rpc) image  on DUT built with SONIC_DEBUGGING_ON=y and INSTALL_DEBUG_TOOLS=y
```
root@arc-switch1025:/# dpkg --list | grep dbg

ii  libsaimetadata-dbg          1.0.0                          amd64        debugging symbols for libsaimetadata
ii  libsairedis-dbg             1.0.0                          amd64        debugging symbols for libsairedis
ii  libswsscommon-dbg           1.0.0                          amd64        debugging symbols for libswsscommon library.
ii  syncd-rpc-dbg               1.0.0                          amd64        debugging symbols for syncd-rpc

```

**- Description for the changelog**
[buildsystem] Install debug packages in syncd when INSTALL_DEBUG_TOOLS=y

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by yxieca** [comment]
I don't think all SAI comes with dbg packet available. These SAI open sourced and built with sonic-buildimage will more likely have access to the dbg packet. But not others, e.g. Broadcom SAI doesn't have dbg packet available. You might want to remove this feature from these flavors of SAI.

**Comment by stepanblyschak** [comment]
@yxieca This change does not install dbg packages for SAI vendor implementation;
libsairedis-dbg and libsaimetadata-dbg come from sairedis sources

**Comment by stepanblyschak** [comment]
retest this please

**Comment by stepanblyschak** [comment]
retest this please

**Comment by lguohan** [comment]
@renukamanavalan , can you give your comments here?

File: platform/barefoot/docker-syncd-bfn-rpc/Dockerfile.j2 [hunk 1]
```diff
@@ -9,7 +9,7 @@ debs/{{ deb }}{{' '}}
 {%- endfor -%}
 debs/
 
-RUN dpkg -P syncd
+RUN apt-get remove -y syncd
```
Comment (qiluo-msft) [inline platform/barefoot/docker-syncd-bfn-rpc/Dockerfile.j2]: Use `apt-get purge`?

File: platform/broadcom/docker-syncd-brcm-rpc/Dockerfile.j2 [hunk 2]
```diff
@@ -9,7 +9,7 @@ debs/{{ deb }}{{' '}}
 {%- endfor -%}
 debs/
 
-RUN dpkg -P syncd
+RUN apt-get remove -y syncd
```
Comment (qiluo-msft) [inline platform/broadcom/docker-syncd-brcm-rpc/Dockerfile.j2]: The same

File: platform/cavium/docker-syncd-cavm-rpc/Dockerfile.j2 [hunk 3]
```diff
@@ -9,7 +9,7 @@ debs/{{ deb }}{{' '}}
 {%- endfor -%}
 debs/
 
-RUN dpkg -P syncd
+RUN apt-get remove -y syncd
```
Comment (qiluo-msft) [inline platform/cavium/docker-syncd-cavm-rpc/Dockerfile.j2]: The same

**Comment by lguohan** [comment]
purge is cleaner.

**Pull request merged**